candy.bundle.js

```diff
@@ -241,10 +241,10 @@ Candy.Core = function(self, Strophe, $) {
             _connection.connect(_getEscapedJidFromJid(jidOrHost) + "/" + _options.resource, null, Candy.Core.Event.Strophe.Connect);
             _user = new self.ChatUser(null, nick);
         } else if (jidOrHost) {
-            Candy.Core.Event.Login(jidOrHost);
+            Candy.Core.Event.Login(null, jidOrHost);
         } else {
             // display login modal
-            Candy.Core.Event.Login();
+            Candy.Core.Event.Login(null, null);
```

Comment:
It's not actually necessary to pass these options - they'll come out as `undefined` if you leave them off the method invocation.